### DISCUSSION TITLE
新聞裡的真實性

### DISCUSSION THREAD

**Discussion by JessyCheng2003** [1101-GS4719-C/Python-Community-GS4719-C#1420] [General]
有看到其他同學有發表關於媒體試讀能力和新聞的真實性，讓我想到之前在大眾傳播課有提到的三種真實性—社會真實、媒介真實、主觀真實。

這三種真實性互相關聯，同時有各自的特點和局限性。在媒介傳播和社會互動中，我們需要辨別不同層面的真實性，以便更好地理解事物的本質和特點。
(1) 社會真實:
指現實生活中的真實情況，包括社會結構、文化、歷史、政治、經濟等方面的真實性。社會真實是客觀存在的，不受個人主觀意識的影響。
(2) 媒介真實:
指媒體呈現的事物與現實生活中的真實情況的關係。媒介真實是相對的，它受到媒體的選擇、呈現方式、語言等方面的影響。媒介的真實性受到諸多因素的制約，如編輯政策、商業利益、主管機構等。
(3) 主觀真實:
指個人對事物的真實感受和體驗。主觀真實是建立在個人主觀意識的基礎上，受到個人的價值觀、經驗、情感等因素的影響。

我認為這是一個很有趣的主題，它讓我深刻意識到事情的真實並不是一個單一的概念，而是有不同的層面。

**Comment by Alexgns** [comment]
我認為當今大部分媒體經常採用單一主觀立場來做報導，因此在媒體真實中可能只包含部分的社會真實，而主觀真實又會受到經常接觸的環境所影響，的確是環環相扣，若要真正了解社會真實，可能就必須多方參考，才不會受到特定人士操弄

**Comment by alicia0725** [comment]
我認為當今社會中的新聞大多都有代考究，多數新聞都增添了不少電視台、主播各人觀點在內，尤其是娛樂、自殺類似的新聞，多數都被渲染的很離譜，常常只是一張照片就被延伸出許多額外的意義

**Comment by ALAN30359704** [comment]
我覺得現今的媒體很多資料，來自於記者本身對於事件的看法，也因為記者要下聳動的標題才能製造較高的點閱率，導致民眾在讀取新聞的時候常常會被聳動的標題所影響，對於事件抱有刻板印象。

**Comment by ashleyy-huang** [comment]
我認為現在的記者發文都有失中立性，許多人只是把自己的想法打出來就按下傳送出去了，這樣很容易亂帶風向或者是提供假新聞給大眾，甚至是為了政黨分派而說了許多不是事實的事，所以培養媒體識讀的能力真的很重要

**Comment by larry40305** [comment]
在面對新聞和媒體報導時，我們應該保持批判思考，不僅僅接受表面的呈現，而是思考背後的動機和可能的影響。同時，我們也需要意識到自己的主觀觀點和經驗，以更客觀的角度來評估信息的真實性。

**Comment by bestbanana295** [comment]
我覺得新聞的標題有時候為了博仁眼球都會故意製造很幒動的標題來引人視聽，因為有了流量他們才可以更好的運作，所以最好的方式就是不要去點閱，用心去查證。

**Comment by ThanatosJun** [comment]
社會真實、媒介真實、主觀真實，我覺得這三種真實各有關聯，其實就是個循環。社會真實影響了我們的主觀真實，媒體製作者也因主觀真實影響了媒介真實，而新聞等各類資訊也影響了社會真實。這三相之力我認為就是個無限迴圈。

**Comment by BOSENCHANG2004** [comment]
為了搶快、搶頭條，很多媒體沒經過完整查證就先發，出錯後再默默更正，甚至根本不更正。有些新聞標題故意誇張或斷章取義，讓你點進去才發現內容完全不一樣，這就是所謂的「釣魚新聞」。媒體常有明顯的政治立場或財團背景，導致同一事件會被報導成兩種完全不同的版本。
你看藍的媒體一套、綠的媒體又是另一套。很多人不再讀正式新聞，而是從 IG、FB、TikTok 上滑到一則「看起來像新聞」的貼文就相信。問題是：這些資訊常常沒來源、沒查證，甚至是刻意操縱。

**Comment by XiangLuen** [comment]
媒體識讀真的很重要，現在媒體都不查證，只要給錢他就會幫你做新聞，也會用各種吸引人的標題片你進去觀看，然後才會發現內容沒人想看，像是最近就有看到andy住信義區高級豪宅和與富家千金新歡，新聞完全就是在帶風向，但就是會有人不知情而相信媒體

**Comment by SendaiHatsuki** [comment]
聽過一個很有道理的句子，「不看新聞，你會跟世界脫節。看了新聞，你會跟真相脫節。」現今媒體為了搏點閱和流量，常常未經查證就發布，或者常常利用醒目的釣魚標題來騙人，在這種快節奏的社會，我們需要花在媒體識讀上的時間就應該越多，避免被帶風向。
